From fbcfe983cdeb723420d67364022fc91ac8c6be3c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alejandro=20S=C3=A1ez?= <asm@redhat.com>
Date: Wed, 5 Jun 2024 12:30:46 +0200
Subject: [PATCH] Modify environment variables defaults

- Set GOTOOLCHAIN to local
- Set GOAMD64 to v3
- Set GOPPC64 to power9
---
 go.env | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/go.env b/go.env
index 6ff2b921d4..0bff84446c 100644
--- a/go.env
+++ b/go.env
@@ -9,4 +9,10 @@ GOSUMDB=sum.golang.org
 
 # Automatically download newer toolchains as directed by go.mod files.
 # See https://go.dev/doc/toolchain for details.
-GOTOOLCHAIN=auto
+GOTOOLCHAIN=local
+
+# The AMD64 baseline for RHEL10 is v3.
+GOAMD64=v3
+# The PPC64LE baseline for RHEL10 is power9
+GOPPC64=power9
+
-- 
2.45.1

